src/components/Header.js

```diff
@@ -96,6 +97,8 @@ class Header extends Component {
       title = titleOfStory
```

Comment:
As mentioned previously I think you need to covert this into a function with returns in a switch. Happy if you do it in another PR.